astro-migration-provider/README.md

```diff
@@ -0,0 +1,60 @@
+# Astronomer Migration Provider
+
+Apache Airflow Provider containing Operators from Astronomer. The purpose of these operators is to better assist customers migrating to Astronomer hosted Airflow environments from MWAA, GCC, OSS
+
+## Installation
+Install and update using [pip](https://pip.pypa.io/en/stable/getting-started/):
+```text
+pip install https://astro-migration-provider.s3.us-west-2.amazonaws.com/astronomer-migration-provider-0.1.0.tar.gz
+```
+
+## Usage
+1. Add the following line to your `requirements.txt` in your source environment:
+   ```text
+    pip install https://astro-migration-provider.s3.us-west-2.amazonaws.com/astronomer-migration-provider-0.1.0.tar.gz
+    ```
+2. Add the following DAG to your source environment:
+    ```python
+    from airflow import DAG
+    from datetime import datetime
+    
+    from astronomer.migration.operators import (
+        AstroVariableMigrationOperator,
+        AstroConnectionsMigrationOperator,
+        AstroEnvMigrationOperator
+    )
+    
+    with DAG(
+        dag_id="astronomer_migration_dag",
+        start_date=datetime(2020, 8, 15),
+        schedule_interval="@once"
+    ) as dag:
+    
+        variables = AstroVariableMigrationOperator(
+            task_id='export_variables',
+            deployment_url='{{ var.value.deployment_url }}',
+            token='{{ var.value.astro_token }}',
```

Comment:
Can this be `{{ params. ... }}` instead? In the spirit of trying to keep changes or setup required to an absolute minimum

Reply:
Not sure by what you mean here? Where would the token be stored if it was a param?

Reply:
followed up in slack, recording here - mentioned [DAG params](https://airflow.apache.org/docs/apache-airflow/stable/concepts/params.html), mentioned that both work for the purposes of the spike

Reply:
params isn't available in all Airflow 2.x versions. Defaulting to airflow variable for first iteration